lib/dbservice_web/router.ex

```diff
@@ -63,6 +63,7 @@ defmodule DbserviceWeb.Router do
     patch("/update-user-enrollment-records", StudentController, :update_user_enrollment_records)
     post("/student/batch-process", StudentController, :batch_process)
     post("/group-user/batch-process", GroupUserController, :batch_process)
+    resources("/college", CollegeController)
```

Comment:
```suggestion
    resources("/college", CollegeController, except: [:new, :edit])
```
Excluding `:new` and `:edit` as we're building a JSON API and don't require server-rendered HTML forms for creating or editing colleges.